extensions/tokenizers/src/main/java/ai/djl/engine/rust/RustLibrary.java

```diff
@@ -22,7 +22,7 @@ private RustLibrary() {}
 
     public static native boolean isCudaAvailable();
 
-    public static native long loadModel(String modelPath, int dtype);
+    public static native long loadModel(String modelPath, int dtype, String device);
```

Comment:
Why we use String, can we just use device id?

Reply:
How distinguish cpu and cuda?

Reply:
https://github.com/deepjavalibrary/djl/blob/master/engines/pytorch/pytorch-engine/src/main/java/ai/djl/pytorch/jni/JniUtils.java#L1785

Reply:
```
public static native long loadModel(String modelPath, int dtype, int deviceType, int deviceId);
```

Reply:
Done.